php/php.refactoring/src/org/netbeans/modules/refactoring/php/findusages/PhpWhereUsedQueryPlugin.java

```diff
@@ -63,6 +64,10 @@ public Problem prepare(final RefactoringElementsBag elementsBag) {
                 if (fileObject == null) {
                     continue;
                 }
+                if (cancelled) {
+                    cancelled = false;
```

Comment:
Maybe, need not this. `cancelled` is used only here at the moment, but I think we should keep the cancelled state.

Reply:
If state is left, then it is impossible to repeat action by clicking green arrows in Usages window.

Reply:
Ah, I see. You are right. Thanks.